packages/gatsby-source-filesystem/src/__tests__/utils.js

```diff
@@ -22,3 +22,13 @@ describe(`create remote file node`, () => {
     })
   })
 })
+
+describe(`slash path`, () => {
+  it(`can correctly slash path`, () => {
+    ;[[`foo\\bar`, `foo/bar`], [`foo\\中文`, `foo/中文`]].forEach(
```

Comment:
remove unnecessary semicolon

```suggestion
    [[`foo\\bar`, `foo/bar`], [`foo\\中文`, `foo/中文`]].forEach(
```